dev/devicelab/manifest.yaml

```diff
@@ -104,7 +104,13 @@ tasks:
       Measures the runtime performance of the Complex Layout sample app on
       Android.
     stage: devicelab
-    required_agent_capabilities: ["mac/android"]
+    required_agent_capabilities: ["linux/android"]
+
+  tiles_scroll_perf__timeline_summary:
+    description: >
+      Measures the runtime performance of the tiles tab in the Complex Layout sample app on Android.
+    stage: devicelab
+    required_agent_capabilities: ["linux/android"]
```

Comment:
linux/android is a bit overloaded compared to mac/android, might be worth putting these on mac/android instead.

Please make sure to set flaky:true when first landing a new test.

Reply:
Makes sense. We could get more linux/android agents too.